index.html

```diff
@@ -7,6 +7,10 @@
     <title>HHI Map</title>
     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.0/dist/leaflet.css" />
     <link rel="stylesheet" href="/src/leaflet-search.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
These look good! I think the current project downloads a bunch of these typography to ship with the app; we can delete all those files now that we have these.